onebusaway-android/src/main/java/org/onebusaway/android/nav/NavigationServiceProvider.java

```diff
@@ -32,17 +25,32 @@
 import android.speech.tts.TextToSpeech;
 import android.util.Log;
 
+import org.onebusaway.android.R;
+import org.onebusaway.android.app.Application;
+import org.onebusaway.android.nav.model.Path;
+import org.onebusaway.android.nav.model.PathLink;
+import org.onebusaway.android.ui.FeedbackActivity;
+import org.onebusaway.android.ui.TripDetailsActivity;
+import org.onebusaway.android.util.RegionUtils;
+
 import java.text.DecimalFormat;
 import java.util.Locale;
 
 import androidx.core.app.NotificationCompat;
+import androidx.core.app.RemoteInput;
 
 /**
  * This class provides the navigation functionality for the destination reminders
  */
 public class NavigationServiceProvider implements TextToSpeech.OnInitListener {
 
     public static final String TAG = "NavServiceProvider";
+    public static String REPLY_ACTION = "org.onebusaway.android.nav.REPLY_ACTION";
```

Comment:
This should be moved to `FeedbackReceiver` and should be declared as:

public static final String ACTION_REPLY =
            BuildConfig.APPLICATION_ID + ".action.REPLY";

Reply:
Added public static final String ACTION_REPLY to FeedbackReceiver